packages/compass-generative-ai/src/utils/gen-ai-prompt.ts

```diff
@@ -0,0 +1,178 @@
+// When including sample documents, we want to ensure that we do not
+// attach large documents and exceed the limit. OpenAI roughly estimates
+// 4 characters = 1 token and we should not exceed context window limits.
+// This roughly translates to 128k tokens.
+const MAX_TOTAL_PROMPT_LENGTH = 512000;
```

Comment:
This limit was for 4o, which had a token limit of 128000. Is that about the limit of what the chatbot api allows?

Reply:
Nice. We could add a TODO in the code, fine as is now though.